src/frontend/context/BrevModulContext.ts

```diff
@@ -247,6 +256,24 @@ const [BrevModulProvider, useBrevModul] = createUseContext(() => {
         avhengigheter: { brevmal },
     });
 
+    const mottakerlandSed = useFelt<string>({
+        verdi: '',
+        valideringsfunksjon: (felt: FeltState<string>) => {
+            return felt.verdi === 'NO'
+                ? feil(felt, 'Norge kan ikke velges som mottakerland')
+                : !isEmpty(felt.verdi)
+                ? ok(felt)
+                : feil(felt, 'Velg land SED er sendt/skal sendes til');
```

Comment:
Er det mulig å skrive om dette til if-else i stedet for ternary? Syns det er litt lettere å lese (og stemmer overens med retningslinjer https://github.com/navikt/familie) 😄